src/qibojit/tests/conftest.py

```diff
@@ -15,7 +25,10 @@
 
 
 @pytest.fixture
-def backend(backend_name):
+def backend(backend_name, request):
+    if request.config.getoption("--gpu_only"):  # pragma: no cover
```

Comment:
Ok, this `# pragma: no cover` is now ubiquitous...

Maybe we should take into account to extract tests from the package even in Qibojit...